lint

```diff
@@ -18,18 +18,13 @@
 set -e
 
 IGNORE_LINT_COMMENT=
-IGNORE_TEST_PACKAGES=
 IGNORE_SPELLINGS=
 while true; do
     case "$1" in
         -nocomment)
             IGNORE_LINT_COMMENT=1
             shift 1
             ;;
-        -notestpackage)
```

Comment:
You might want to consider leaving this in as a noop, as the next time weave/cortex is updated this will break.  Or not, and update the repos yourself...